docs/extras/wss.md

```diff
@@ -0,0 +1,13 @@
+# WSS (WebSocket) Node
+
+The WSS extra node can be used by adding the following to `pyproject.toml`
+
+:   
+    ```yaml title="pyproject.toml" hl_lines="2"
+    [tool.poetry.dependencies]
+    aineko = {version = "^0.2.7", extras=["wss"]}
+    ```
```

Comment:
It might be better to tell the user to do `poetry add aineko[wss]`.
That command will take care of updating the pyproject.toml, updating the poetry.lock and installing the new dependencies.

Reply:
@craycrayfish any reason you did not do it the way @Barabazs suggests for FastAPI?